Defs/Bodies/BodyShapes.xml

```diff
@@ -12,6 +12,14 @@
 		<height>1</height>
 		<heightLaying>0.3</heightLaying>
 	</CombatExtended.BodyShapeDef>
+	
+	<CombatExtended.BodyShapeDef>
```

Comment:
What kind of creatures do you see this having an application for?

Reply:
dwarfs